SPECS/dracut/dracut.spec

```diff
@@ -113,6 +113,7 @@ This package contains tools to assemble the local initrd and host configuration.
 %package overlayfs
 Summary:        dracut module to build a dracut initramfs with OverlayFS support
 Requires:       %{name} = %{version}-%{release}
+Conflicts:      %{name}-xen <= 102-4
```

Comment:
Maybe add a quick comment here so we can easily clean this up in the future if we do a major version bump.

Reply:
I guess since all subpackages Requires:       %{name} = %{version}-%{release}, it is already enforced that overlayfs-102-5 can't be on the same system as xen-102-4. Maybe we just drop the Conflicts entirely.